include/motor-control/core/motor_driver.hpp

```diff
@@ -34,12 +42,12 @@ class MotorDriver {
     MotorDriver(SpiDriver& spi) : spi_comms(spi) {}
 
     void setup() {
-        // GCONF 0x01
-        // IHOLD_IRUN 0x1010
-        // CHOPCONF 0x8008
-        constexpr uint32_t gconf_data = 0x01;
-        constexpr uint32_t ihold_irun_data = 0x1010;
-        constexpr uint32_t chopconf = 0x8008;
+        constexpr uint32_t gconf_data = 0x04;
```

Comment:
These values at some point need some documenting

Reply:
They're still pretty influx, but will definitely document once we have solidified setup values.